### ISSUE TITLE
Selection of (six) featured articles for the release

### ISSUE THREAD

**Issue by sefeg** [cernopendata/opendata.cern.ch#1732]
While the functionality to mark posts as featured is being implemented, we can already collect a set  that we want to feature in the upcoming release. We can feature **six** on the landing page. Here, we can also discuss the position of those. For this issue, I propose to consider the top left post on the landing page as number 1, the right bottom one as number 6.

The following represents a list of contents we discussed so far. An element should be checked as soon as the article is publish-ready (even though it might still need editing). I will directly link all those that have been added to COD. That allows everyone to review and edit the contents. (**Edit:** Lists updated 7 December to reflect current priorities.) 

**Feature these six on the homepage**
- [x] [Welcome to our updated portal ](http://opendata-dev.web.cern.ch/docs/welcome-to-our-updated-portal)
- [ ] CMS 2012 data release
- [x] [Jet substructure paper by MIT theorists](http://opendata-dev.web.cern.ch/docs/cms-the-future-is-open-2017)
- [x] [Guide to education use of CMS Open Data](http://opendata-dev.web.cern.ch/docs/cms-guide-to-education-use-of-cms-open-data)
- [ ] LHCb AntimatterMatters2017
- [x] [ATLAS Higgs Machine Learning challenge](http://opendata-dev.web.cern.ch/docs/atlas-higgs-machine-learning-challenge)

**Additional provided content**
- [x] [Improving educational content with high school teachers: A summer student report](http://opendata-dev.web.cern.ch/docs/improving-educational-content-with-high-school-teachers-a-field-report-from-our-summer-students)
- [x] [Guide to research use of CERN Open Data](http://opendata-dev.web.cern.ch/docs/cms-guide-to-research-use-of-cms-open-data) 
- [ ] OPERA open data release - postponed upon request

Towards the end of the sprint / release, we can discuss which of the ready articles finally are featured and which positions they take.

____
My initial thoughts (look at the list above for most current overview):

**1 – Welcome to our updated portal**

With the growing demand of open science data and increased interest in the CERN Open Data portal, we were working hard on modernizing our web presence. Here, we are telling you more about the changes we made and how you can use the updated portal…

![image](https://user-images.githubusercontent.com/25487025/32649774-9541d742-c5fb-11e7-8e41-c6931a814fe3.png)

**2 – CMS 2012 data release**

Image could be like the one that follows, or alternatively a Higgs plot, if a nice one can be produced. (more on that news records from @katilp )

![image](https://user-images.githubusercontent.com/25487025/32649800-a73366c8-c5fb-11e7-81a3-7b37d8677e03.png)

**3 - Improving educational content with high school teachers: A summer student report**

![image](https://user-images.githubusercontent.com/25487025/32649886-0456f0f4-c5fc-11e7-9616-de2ff76e0e9d.png)

**4 – CMS Guide to education use of CERN Open Data**

Image?

**5 – ATLAS Higgs Machine Learning challenge**

Image?

**6 – CMS Guide to research use of CERN Open Data**

Image?

**Comment by tiborsimko** [comment]
There are perhaps too many CMS related items in the above list?

Two other good candidates would be:

- OPERA open data release (we have news statement, and the illustrating screenshot can be taken from the event display)
- LHCb AntimatterMatters2017 (the article to be written with Chris & Chris)

**Comment by sefeg** [comment]
I agree! Since we have a lot of very good CMS guiding material, we can great an umbrella record that links to the various CMS resources. That way, we would only have two featured CMS records in the list. (The summer student record is not that visible as a CMS related resource).

**Comment by katilp** [comment]
We may have the contribution from Jesse Thaler as well, so we need to prioritize from the CMS side. Will there be a "Show more" option to see more "News" articles on the entry page?

**Comment by suenjedt** [comment]
An article by Jesse Thaler would be great, of course, @katilp . 
@sefeg i am not convinced of the umbrella article (if i understand it correctly), and would prefer to include Opera and Lhcb to balance.

**Comment by tiborsimko** [comment]
I would also prefer no umbrellas... and I like the show-more button mentioned by @katilp.  This could simply lead to the search page where we would display all articles, sorted by chronological order,  featured only, facet by experiment, and all the other goodies.

**Comment by suenjedt** [comment]
@tiborsimko @sefeg do you want me to contact our LHCb and Opera contacts to coordinate the featured article? I am aware that we have materials for Opera (still would be good to share plans).

**Comment by sefeg** [comment]
@suenjedt that would be great! 

I think @tiborsimko is on to an article on LHCb AntimatterMatters2017

As soon as an article is ready to be featured, we can make a note in this issue.

**Comment by sefeg** [comment]
The following represents a list of contents we discussed in here so far. An element should be checked as soon as the article is publish-ready (even though it might still need editing). I will change the font style to **bold** for all those that have been added to COD.

- [x] Welcome to our updated portal
- [ ] CMS 2012 data release
- [x] Improving educational content with high school teachers: A summer student report
- [ ] CMS Guide to education use of CERN Open Data
- [ ] CMS Guide to research use of CERN Open Data
- [x] **ATLAS Higgs Machine Learning challenge**
- [ ] OPERA open data release 
- [ ] LHCb AntimatterMatters2017

Towards the end of the sprint / release, we can discuss witch of the ready articles finally are featured and which positions they take.

@artfisica if you like to, you could also use this issue to quickly describe the articles you are writing / planning and I will add them to the list

**Comment by katilp** [comment]
Jesse will provide the story by the end of the month. He already suggested an image for illustration:
[pr_event_display_no_logo.pdf](https://github.com/cernopendata/opendata.cern.ch/files/1462257/pr_event_display_no_logo.pdf) or 
[pr_event_display_with_sub.pdf](https://github.com/cernopendata/opendata.cern.ch/files/1462258/pr_event_display_with_sub.pdf) (with or without their logo)

For CMS, this has a priority (after the CMS 2012 release announcement).
Can we combine the two guides in the list above in one article (to save feature space)

**Comment by RaoOfPhysics** [comment]
@katilp: As I understand it, our priorities are:

1. 2012 release
2. Jet substructure paper by MIT theorists
3. Anything else…

Perhaps for now the first two will suffice?

Also, @sefeg, can you add the check-list + links to the OP of this issue, so we can track progress directly there?

**Comment by katilp** [comment]
@RaoOfPhysics Yes, that's I how I would see it. 3 can be Guide to education use of CMS Open Data
(which has just been updated)

**Comment by sefeg** [comment]
Done! Moved the list to the top of the issue. @RaoOfPhysics

**Comment by RaoOfPhysics** [comment]
@sefeg: Ace, thanks. :)

**Comment by sefeg** [comment]
"Welcome to our updated portal" is now [available on dev](http://opendata-dev.web.cern.ch/articles/welcome-to-our-updated-portal). @daslerr @suenjedt please feel free to edit the article
The markdown file can be found [here](https://github.com/cernopendata/opendata.cern.ch/blob/master/cernopendata/modules/fixtures/data/articles/welcome/welcome.md).

**Comment by sefeg** [comment]
Absolutely! Any support is highly appreciated @RaoOfPhysics thank you!

**Comment by suenjedt** [comment]
@RaoOfPhysics I continue once you are done. We need to remove Opera from this text though.

**Comment by RaoOfPhysics** [comment]
@suenjedt: Thanks, I can remove OPERA. Anything specific I should do with it? I'm in the middle of some other work, but I should be able to submit a PR by lunch.

Also, can I change the author to "CERN Open Data team" rather than "Open Data Portal team"? Thanks!

**Comment by suenjedt** [comment]
@RaoOfPhysics 
Yes, please change to CERN Open Data team. 
And just take Opera out. Will be released with a separate announcement.

**Comment by daslerr** [comment]
I updated the cheklist at the top to reflect the priorities discussed in the comments and to include the MIT article. @RaoOfPhysics are you in charge of the CMS 2012 data release announcement?

**Comment by ioannistsanaktsidis** [comment]
As I see on the checkbox list we already have 6 records to display in the homepage. Is this one considered as done ? cc @daslerr @sefeg

**Comment by suenjedt** [comment]
We would like to wait for the lhcb contribution which shall arrive beginning of next week.

**Comment by daslerr** [comment]
I've updated the checklists to be more specific. :) Yes, we're waiting on the LHCb contribution and the final CMS 2012 announcement.

**Comment by RaoOfPhysics** [comment]
@daslerr: Yes, I'm drafting the CMS 2012 release announcement.

**Comment by tiborsimko** [comment]
Three articles selected, completed, and deployed. Closing the issue.
